CHANGES.md

```diff
@@ -437,6 +437,10 @@ OpenSSL 3.0
 
    *Antonio Iacono*
 
+ * Added the AuthEnvelopedData (RFC 5083) with AES-GCM parameter (RFC 5084).
```

Comment:
Could you  please expand the text a little? Add something like `Added ... support to the CMS implementation.` Or something like that to make it clear without having to look into the RFCs or code where the support was added.

Reply:
@t8m Makes sense. I have extended the text a bit. Please let if you want me to change anything in it.